api/responseMapper.go

```diff
@@ -2,8 +2,11 @@ package api
 
 import "github.com/supplyon/gremcos/interfaces"
 
+// ResponseArray an array type for responses
 type ResponseArray []interfaces.Response
 
+// ToValues converts the given ResponseArray into an array of TypedValue type.
+// The method will fail in case the data in the given ResponseArray are no primitive types.
```

Comment:
Please rephrase: "The method will fail in case the data in the given ResponseArray is not a primitive type"